app/src/main/java/de/tum/in/tca/ticketcheck/component/ticket/AdminDetailsActivity.java

```diff
@@ -0,0 +1,138 @@
+    private TextView totalSaleView;
+    private FloatingActionButton floatingScanner;
+    private TicketListAdapter mAdapter;
+    private TicketListAdapter findAdapter;
+    private List<AdminTicket> tickets;
+    private List<AdminTicket> findList;
+    private MenuItem menuItemScanView;
+
+    public AdminDetailsActivity() {
+        super(R.layout.activity_admin);
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        searchView = findViewById(R.id.searchTicket);
+        totalSaleView = findViewById(R.id.total_sale);
+        listView = findViewById(R.id.listTicket);
+        floatingScanner = findViewById(R.id.fab_scanner);
+
+        int clickedEventId = getIntent().getIntExtra("event_id", 0);
+        //TODO:send event_id to backend and receive really ticket data,following just use dummy ticketdata
+        tickets = TicketsController.getTickets();
+
+        findList = new ArrayList<>();
+        mAdapter = new TicketListAdapter(tickets, this);
+        listView.setAdapter(mAdapter);
+
+        //Set total ticket and total sale
+        //TODO:send event_id to backend and receive really sum of ticket,following just use dummy ticketdata
+        int totalTicketNumber = 100;
+        int totalSaleNumber = tickets.size();
+        String totalSaleString = "Total Sale : " + totalSaleNumber + "/" + totalTicketNumber;
+        totalSaleView.setText(totalSaleString);
+
+        //Set FAB to scanner
+        floatingScanner.setOnClickListener(view -> scanning());
+        // Set search function
+        setupSearchView();
+
+    }
+
+    public boolean onCreateOptionsMenu(Menu menu) {
+        super.onCreateOptionsMenu(menu);
+        getMenuInflater().inflate(R.menu.menu_activity_admin, menu);
+        menuItemScanView = menu.findItem(R.id.action_refresh);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        int i = item.getItemId();
+        if (i == R.id.action_refresh) {
+            //TODO:send event_id to backend and receive really ticket data,following just use dummy ticketdata
+            tickets = TicketsController.getrefreshTickets();
+            mAdapter = new TicketListAdapter(tickets, this);
+            listView.setAdapter(mAdapter);
+            mAdapter.notifyDataSetChanged();
+        }
+        return true;
+    }
+
+    private void scanning() {
+        startActivity(new Intent(this, TicketScanActivity.class));
+        //TODO:send current eventId to TicketScanActivity
+    }
+
+    private void setupSearchView() {
+        searchView.setSubmitButtonEnabled(true);
+        searchView.setOnQueryTextListener(new SearchView.OnQueryTextListener() {
+            public boolean onQueryTextSubmit(String query) {
+                if (TextUtils.isEmpty(query)) {
+                    Toast.makeText(AdminDetailsActivity.this, "please text something", Toast.LENGTH_SHORT).show();
```

Comment:
I think we should use Utils.showToast
Also add the strings to strings.xml

Reply:
Done it